handlers/config.go

```diff
@@ -51,6 +55,7 @@ type PublicConfig struct {
 	Deployment          DeploymentConfig              `json:"deployment,omitempty"`
 	GatewayAPIClasses   []config.GatewayAPIClass      `json:"gatewayAPIClasses,omitempty"`
 	GatewayAPIEnabled   bool                          `json:"gatewayAPIEnabled,omitempty"`
+	Grafana             GrafanaConfig                 `json:"grafana,omitempty"`
```

Comment:
This is placed in the server config, but this is only used for Grafana, right? Shouldn't be better to place it in the grafana config api call? 
![image](https://github.com/user-attachments/assets/86f66bb3-6fbf-432a-8615-ea85bc9bf2da)